lib/virtfs/iso9660/boot_sector.rb

```diff
@@ -0,0 +1,167 @@
+require 'sys-uname'
+
+module VirtFS::ISO9660
```

Comment:
Hmmm, how does this module relate to the `Iso9660` module I commented on earlier?